engine/src/test/java/io/camunda/zeebe/streamprocessor/RecordBatchTest.java

```diff
@@ -202,4 +202,74 @@ public void shouldOnlyAddUntilMaxBatchSizeIsReached() {
         .hasMessageContaining("Can't append entry")
         .hasMessageContaining("[ currentBatchEntryCount: 1, currentBatchSize: 249]");
   }
+
+  @Test
+  public void shouldReturnFalseIfRecordSizeDoesReachSizelimit() {
```

Comment:
:wrench: Since these are junit5 tests, they can be not public.

Reply:
Thanks wasn't aware.

Just checked https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods it seems it is recommended to remove it, but it is not really an issue.

> It is generally recommended to omit the public modifier for test classes,

I will remove them, thanks. :)